tests/contrib/test_gce.py

```diff
@@ -66,11 +66,13 @@ def _refresh_success_helper(self, bytes_response=False):
         credentials.refresh(http)
         self.assertEquals(access_token, credentials.access_token)
 
-        base_metadata_uri = ('http://metadata.google.internal/0.1/meta-data/'
-                             'service-accounts/default/acquire')
+        base_metadata_uri = (
```

Comment:
Why not make use of `gce.META` here? I suppose it's a philosophical question: does the correctness of the `gce` module (which is the matter under test) depend upon this exact URL value, or does it depend upon the `gce` module using whatever value is stored in its `META` attribute?